react_native_fundamentals/index.md

```diff
@@ -1,7 +1,23 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Introduction 
+    - iOS and Android overview
+    - Types of apps
+    - What is React Native
+    - Expo vs React Native
+    - Alternatives 
+2. Environment setup
+3. Creating a new app
+4. Project structure overview
+5. Core Components
```

Comment:
maybe we can go with Components name for this item, since the general overview will include mention of Native Components and Community provided components. Please let me know your thoughts.